portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -406,9 +406,11 @@ if (sessionError != null) {  %>
     <span class='annotation-item chang_hotspot' alt='{{changHotspotAlt}}'>
         <img width='{{hotspotsImgWidth}}' height='{{hotspotsImgHeight}}' src='{{hotspotsImgSrc}}' alt='Recurrent Hotspot Symbol'>
     </span>
+    <% if (showCivic) { %>
```

Comment:
The underscore template definition would be server-side independent if we use an underscore condition check instead of JSP condition check.

Having said that we won't be using underscore templates after refactoring, so we can handle this while rewriting with React.